components/adblock_rust_ffi/Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Brian R. Bondy <netzen@gmail.com>"]
 edition = "2018"
 
 [dependencies]
-adblock = { version = "0.6.0", default-features = false, features = ["full-regex-handling", "object-pooling", "unsync-regex-caching"] }
+adblock = { version = "0.6.1", default-features = false, features = ["full-regex-handling", "object-pooling", "unsync-regex-caching"] }
```

Comment:
@brave/rust-deps-reviewers the only change between `v0.6.0` and `v0.6.1` is the addition of the `read_list_metadata` function: https://github.com/brave/adblock-rust/commit/89c1a1a71a6e5c4791b820ed6a0e5f39bb09de20